ONSdigital/design-system
# Fix webkit-details-marker warning

### What is the context of this PR?
Fixes: #1534 

### How to review
- Go to https://ons-design-system.netlify.app/
- See the warning in the console
- Go to PR netlify preview
- See warning gone from console
